### DISCUSSION TITLE
Lucid: filtering data

### DISCUSSION THREAD

**Discussion by rbartholomay** [adonisjs/core#1315] [Help]
Hi there,

currently i try to migrate a small project to Adonis5 and cannot figure out how to filter data in the new Version. In Adonis4 i can do this: [AdonisJS/docs/4.1: has](https://adonisjs.com/docs/4.1/relationships#_has)

```
const posts = await Post
  .query()
  .has('comments')
  .fetch()
````

How must i do this in version 5?

Bye...

**Comment by thetutlage** [comment]
Yes, its pending and part of the roadmap

**Reply by addanfelipe** [reply]
(Google translator)

Hello. Is there a forecast (or alternative) for this item?
I'm starting a project and I want to use Adonis v5, but this item is essential for the application.

**Reply by thetutlage** [reply]
Will be out in the next release (first week of August)
